composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerScreen.kt

```diff
@@ -106,59 +120,90 @@ fun IntervalsList() {
 }
 
 @Composable
-fun RepsButton(reps: Int,addReps: () -> Unit,removeReps:() ->Unit) {
-    Column(horizontalAlignment = Alignment.CenterHorizontally,
-        verticalArrangement = Arrangement.Center){
-
-    Box(Modifier.height(43.dp).width(100.dp).background(MaterialTheme.colorScheme.primary),contentAlignment = Alignment.Center){
-        Text("Reps", color = Color.White)
-    }
-        Spacer(Modifier.height(20.dp))
-        Text("+", Modifier.clickable{addReps()},fontSize = 40.sp, color = MaterialTheme.colorScheme.primary )
-        Text("$reps", fontSize = 40.sp,color= MaterialTheme.colorScheme.primary)
-        Text("-",Modifier.clickable{removeReps()}, fontSize = 50.sp,color=MaterialTheme.colorScheme.primary)
-
-    }
+fun RepsButton(reps: Int, addReps: () -> Unit, removeReps: () -> Unit) {
+    TableCard(
+        header = "Reps",
+        content = {
+            Column(
+                horizontalAlignment = Alignment.CenterHorizontally,
+                verticalArrangement = Arrangement.Center,
+            ) {
+                IconButton(
+                    onClick = addReps,
+                ) {
+                    Icon(
+                        imageVector = Icons.Default.Add,
+                        contentDescription = "Add rep",
+                    )
+                }
 
+                Text(
+                    color = MaterialTheme.colorScheme.secondary,
+                    text = "$reps",
+                    style = MaterialTheme.typography.headlineMedium,
+                    fontWeight = FontWeight.Black,
+                )
+
+                IconButton(
+                    onClick = removeReps,
+                ) {
+                    Icon(
+                        imageVector = Icons.Default.Remove,
+                        contentDescription = "Remove rep",
+                    )
+                }
+            }
+        }
+    )
 }
 
 @Composable
-fun AddButton(clickAdd: () -> Unit) {
-    Box(
-        Modifier.height(50.dp).width(200.dp)
-            .clip(RoundedCornerShape(size = 12.dp))
-            .background(MaterialTheme.colorScheme.primary)
-            .clickable { clickAdd() },
-        contentAlignment = Alignment.Center
-    )
-    {
-        Text("Add+", fontSize = 22.sp, color = Color.White)
-
+private fun AddButton(
```

Comment:
Likewise, everything in this function should be `private` except for `TimerScreen`.